modules/replication/module.py

```diff
@@ -69,6 +69,14 @@ def __init__(self, id: int, resolver, n, f, k):
                     self.rep = rep
                 if byz.is_byzantine():
                     self.byz_rep = deepcopy(rep[self.id])
+                    if byz.get_byz_behavior() == byz.WRONG_CCSP:
+                        byz_applied_req = {REQUEST: Request(
+                                            rep[self.id].get_pend_reqs()[1],
+                                            0,
+                                            3),
+                                           X_SET: {0, 1, 2, 3, 4, 5}}
+                        self.byz_rep.set_rep_state([2])
```

Comment:
Same here, try to build `byz_rep.rep_state` to be invalid based on the current state. For example:
```
valid_state = self.byz_rep.get_rep_state()
byz_state = valid_state[0] if len(valid_state) > 1 else [valid_state[0] + 1]
self.byz_rep.set_rep_state(byz_state)
```